CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+# dbt_mixpanel v0.9.0
+[PR #41](https://github.com/fivetran/dbt_mixpanel/pull/41) includes the following updates:
+
+## 🚨 Breaking Changes 🚨
+> ⚠️ Since the following changes are breaking, we recommend running a `--full-refresh` after upgrading to this version.
+- Performance improvements:
+  - Updated the incremental strategy for of the following models to `insert_overwrite` for BigQuery and Databricks and `delete+insert` for all other warehouses. 
+    - `stg_mixpanel__user_event_date_spine`
+    - `mixpanel__event`
+    - `mixpanel__daily_events`
+    - `mixpanel__monthly_events`
+    - `mixpanel__sessions`
+  - Removed `stg_mixpanel__event_tmp` in favor of ephemeral model `stg_mixpanel__event`. This is to reduce redundancy of models created and reduce the number of full scans.
+  - Updated the materialization of `stg_mixpanel__user_first_event` to a view. 
```

Comment:
Updating to:
```suggestion
  - Updated the materialization of `stg_mixpanel__user_first_event` from a table to a view. This model is used in one downstream model, so a view will reduce storage requirements while not significantly hindering performance.
```

Reply:
Sounds great to me, I like this update. Thanks!